dependencies.gradle

```diff
@@ -7,7 +7,7 @@ ext.versions = [
         'targetCompat'      : JavaVersion.VERSION_11,
 ]
 
-def gradle = "7.2.0"
+def gradle = "7.1.3"
```

Comment:
it might be worth a comment here to explain that the version is set to 7.1.3 because of missing coverage / https://github.com/realm/realm-java/issues/7685